CHANGELOG.md

```diff
@@ -12,4 +12,14 @@ Bugfixes:
 
 Other improvements:
 
-## [0.0.0] - 2020-01-01
+## [0.1.0] - 2020-09-21
+This release adds tests for `HugeInt`.
+
+Breaking changes (😱!!!):
+
+New features:
+
+Bugfixes:
+
+Other improvements:
```

Comment:
Could you add this to the [Unreleased] section instead of an existing release? We're going to fill in the old releases from the existing GitHub releases. Something like:

---

```md
## [Unreleased]

Breaking changes (😱!!!):

New features:

Bugfixes:

Other improvements:

- Tests: add tests for `HugeInt` ([#18](https://github.com/purescript-contrib/purescript-precise/issues/18))

## [0.0.0] - 2020-01-01
```

Reply:
oh yeah sure will do that